src/Agent.Listener/JobDispatcher.cs

```diff
@@ -26,6 +26,7 @@ public interface IJobDispatcher : IAgentService
     public sealed class JobDispatcher : AgentService, IJobDispatcher
     {
         private int _poolId;
+        private ITerminal _term;
```

Comment:
you don't actually need a global var _term, since it only been used in one private method.